qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -186,9 +188,10 @@ def forward(self, state: Tensor, _: dict[str, Tensor] = None) -> Tensor:
     def device(self) -> torch_device:
         return self._device
 
-    def to(self, device: torch_device) -> PyQMatrixBlock:
-        self.mat = self.mat.to(device)
-        self._device = device
+    def to(self, *args: Any, **kwargs: Any) -> PyQMatrixBlock:
```

Comment:
Can't we specialise `args` and `kwargs` type annotations ?

Reply:
i think its common practice to use these type annotations

Reply:
Yeah, please ignore. Since we agreed that moving to a `torch` device and letting the possibility choose the precision is up to the user to do it correctly.